package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "npm run lint && karma start --single-run",
     "watchTests": "karma start --reporters nyan",
+    "build": "uglifyjs src/index.js > index.min.js",
```

Comment:
Make the input to uglifyjs `src/*.js` so it automatically captures any new source files you add.